openid-sharedsignals-framework-1_0.md

```diff
@@ -1897,16 +1956,16 @@ subject
   "iss": "https://transmitter.example.com",
   "aud": "receiver.example.com",
   "iat": 1493856000,
-  "events": {
-    "https://schemas.openid.net/secevent/ssf/event-type/stream-updated": {
-      "subject": {
```

Comment:
I think we said that "subject" in the event will be retained but called out as deprecated.